lib/session.js

```diff
@@ -1,16 +1,17 @@
 'use strict'
 
 const Cookie = require('./cookie')
-
+const generateSessionIds = require('./utils')
 const maxAge = Symbol('maxAge')
 
 module.exports = class Session {
-  constructor (sessionId, encryptedSessionId, cookieOpts, prevSession = {}) {
+  constructor (sessionId, encryptedSessionId, cookieOpts, secret, prevSession = {}) {
     this.sessionId = sessionId
     this.encryptedSessionId = encryptedSessionId
     this.expires = null
     this.cookie = new Cookie(cookieOpts)
     this[maxAge] = cookieOpts.maxAge
+    this.secret = secret
```

Comment:
I think it would be good to use a `Symbol` to store the `secret` on the session. It would prevent it being written to the session store if `JSON.stringify()` is being used.

Reply:
ok

Reply:
@SerayaEryn, changes done as requested.